src/java.base/share/classes/java/lang/doc-files/ValueBased.html

```diff
@@ -35,32 +35,37 @@ <h1 id="ValueBased">{@index "Value-based Classes"}</h1>
 <code>java.time.LocalDateTime</code>, are <em>value-based</em>.  Instances of a
 value-based class:
 <ul>
-    <li>are final and immutable (though may contain references to mutable
+    <li>are final</li>
+    <li>have only final instance fields (though these may contain references to mutable
         objects);</li>
+    <li>extend either <code>Object</code> or a hierarchy of abstract classes that
+        declare no instance fields and have no instance initialization logic</li>
+    <li>do not have (or have deprecated for removal any) accessible constructors,
+        but are instead instantiated through factory methods which make no
+        commitment as to the identity of returned instances;</li>
+    <li>expose no identity-sensitive behaviors through their methods, such as by
+        relying on equality (<code>==</code>) between instances, the identity hash
+        code of an instance, or synchronization on an instances's intrinsic lock;</li>
     <li>have implementations of <code>equals</code>,
         <code>hashCode</code>, and <code>toString</code> which are computed
         solely from the instance's state and not from its identity or the state
         of any other object or variable;</li>
-    <li>make no use of identity-sensitive operations such as reference
-        equality (<code>==</code>) between instances, identity hash code of
-        instances, or synchronization on an instances's intrinsic lock;</li>
-    <li>are considered equal solely based on <code>equals()</code>, not
-        based on reference equality (<code>==</code>);</li>
```

Comment:
I removed this bullet because I don't think it really means anything. We already said classes are required not to expose identity through their methods, and to override 'equals'